unit_tests/web_tests/test_web_status.py

```diff
@@ -34,49 +35,57 @@ def __setAliasView(self):
         self.page.click_update_user()
         self.page.click_top_nav('Status')
 
+    '''
```

Comment:
these should no longer be skipped, we will not pull this until the tests are enabled again.